### PR TITLE
Feature/#38 scope support

### PR DESCRIPTION
**Pull request by spyhunter99**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,6 +300,23 @@ $ ./gradlew publish -Pprofile=sources,javadoc,sign
...
+| releaseCompile        | compile
+| debugCompile          | NOT MAPPED*
+| runtime               | runtime
+| testCompile  	        | test   	
+| androidTestCompile    | test
+| provided              | provided
+
+* Items marked as not mapped are not referenced in the pom. Pom's are 
+generally used for releases, such as, items specific to a scope, such as
+'debugCompile' aren't useful since it won't be in the release version.
```
Comment (chrisdoyle) [inline README.md:324]: Good call.

**Comment by spyhunter99** [comment]
it's just not fair! i merged dev back in and now nothing works

**Comment by spyhunter99** [comment]
fun fact, configurations.findAll.name returns a list of all dependencies, specifically, their file names.

and it apparently behaves different for android vs war files

**Comment by chrisdoyle** [comment]
Wait, wait, wait -- something behaves differently for Android?  Get right out of town.

**Comment by chrisdoyle** [comment]
Did you want to add any runtime deps to the test?  If not, then this is a go.

**Comment by spyhunter99** [comment]
fun fact, runtime doesn't exist for android but does for jar projects

**Comment by spyhunter99** [comment]
well congrat's @chrisdoyle, you've discovered another gradle bug.

JAR projects with a build.gradle properties as the following

``` grovy
dependencies {
    compile "org.apache.commons:commons-lang3:${project.property('commons-lang3.version')}"
    runtime 'com.esotericsoftware:kryo:4.0.0'
    testCompile "junit:junit:${project.property('junit.version')}"
}

```

Produces

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.chrisdoyle</groupId>
  <artifactId>hello-world-lib</artifactId>
  <version>1.0.7-SNAPSHOT</version>
  <dependencies>
    <dependency>
      <groupId>com.esotericsoftware</groupId>
      <artifactId>kryo</artifactId>
      <version>4.0.0</version>
      <scope>runtime</scope>
    </dependency>
    <dependency>
      <groupId>org.apache.commons</groupId>
      <artifactId>commons-lang3</artifactId>
      <version>3.4</version>
      <scope>runtime</scope>
    </dependency>
  </dependencies>
  <packaging>jar</packaging>
</project>
```

Note that we're missing name and description, both dependencies are listed as runtime and it's missing all the test dependencies.

**Comment by spyhunter99** [comment]
Update the ticket description, we'll have to cycle back around to this.

The answer is no, runtime will not be included in this PR

**Comment by chrisdoyle** [comment]
"It's a feature!"

**Pull request merged**